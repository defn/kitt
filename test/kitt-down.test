#!/bin/bash

function main {
  kitt down --help
}

main "$@"
