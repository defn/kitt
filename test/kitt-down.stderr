USAGE: kitt down [flags] args

Calls docker-compose down to shut down services.  Services that are not
defined in docker-compose.yml will be removed.

args: arguments to docker-compose

flags:
  -h,--help:  show this help (default: false)
